src/Cache/Redis.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace Kirby\Cache;
+
+use Redis as RedisClient;
+use Throwable;
+
+/**
+ * Redis Cache Driver
+ *
+ * @package   Kirby Cache
+ * @author    Ahmet Bora <ahmet@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://opensource.org/licenses/MIT
+ */
+class Redis extends Cache
```

Comment:
I think the class name should be `RedisCache`, right?

Reply:
Also I would like to use separated namespace for drivers like that:

- Kirby\Cache\Cache
- Kirby\Cache\Value
- Kirby\Cache\Drivers\Apcu
- Kirby\Cache\Drivers\File
- Kirby\Cache\Drivers\MemCached
- Kirby\Cache\Drivers\Memory
- Kirby\Cache\Drivers\Null
- Kirby\Cache\Drivers\Redis

What do you think?

Reply:
Yes it should be `RedisCache`, but why would it need a separate namespace? I think in the end, using something like `new FileCache()` or `new NullCache()` in your code is a little easier than `new File()`/`new Null()`

Reply:
Yes, maybe we will continue to use as `FileCache` like `use Kirby\Cache\Drivers\File as FileCache`. But directory structure will be more readable. Currently mixed drivers and main classes. Not important, just was the idea.

![github com_getkirby_kirby_tree_feature_redis-cache_src_Cache](https://github.com/user-attachments/assets/9260a3b4-9b9d-4ad4-880d-b755f5383e2a)